doc/sslang-lrm.tex

```diff
@@ -328,6 +325,77 @@ \subsection{User-defined operators}
 a look at Agda's user-defined operators.
 
 \section{Types}
+sslang exposes two classes of types. \\
+
+\noindent The first class of types encompasses primitive types: integers, booleans, pure events (i.e. conditional variable), and unit (i.e. void).\\ 
+
+\noindent The second class of types are higher order types that are composed of primitive types:
```

Comment:
This description of type constructors seems to limit us only to the built-in type constructors, and does not permit user-defined type constructors.

Could we generalize and just say something indicating that type constructors are applied to zero or more type arguments, and that they must include the <above> built-in types?

also note that the "primitive" types are just type constructors that take no type arguments. I wouldn't consider them a different "class" of types, but it is still worth giving examples of them to clarify that these types do exist and are quite common/important.